File: week7/Capstone1/todoList.js
module.exports = class Mylist{
    constructor(){
        this.list = []
        this.length = 0
    }

    getList(){
        return Object.values(this.list)
    }

    add(item){
        this.list[this.length] = item;
        this.length++;
        return `${item} has been added`
    }

    multiAdd(words){
        this.list[this.length] = words
        this.length++;
        return `${words} have been added to the list`
       
    }
    removeFirst(item){
        item = this.list.shift()
        this.length--
        return `${item} has been removed from list`
    }
    removeLast(item){
        item = this.list.pop()
        this.length--
        return `${item} has been removed from list`
    }
    removeSpecific(word,item,index){
        index = this.list.indexOf(word)
        item = this.list.splice(word,1)
        this.length--
       return `${item} has been removed from list`
        
    }
}

File: week7/Capstone/todoList.py

class MyTodoList:
    def __init__(self):
        self.list = []
        self.length = 0

    def getList(self):
        return self.list

    def add(self, word):
        self.list.append(word)
        self.length+1
        return f'{word} has been added'

    def addMulti(self, words):
        self.list.extend(words)
        self.length+1
        return self.list

    def removeFirst(self):
        del self.list[0]
        self.length-1
        return self.list

    def removeLast(self):
        del self.list[-1]
        self.length-1
        return self.list

    def removeSpecific(self, word):
        found = self.list.index(word)
        del self.list[found]
        self.length-1
        return self.list
